scripts/generate_anchorage_events

```diff
@@ -88,12 +88,12 @@ fi
 INSERT_SQL=${ASSETS}/bigquery/anchorage-events.sql.j2
 echo "Inserting new records for ${START_DATE} to ${END_DATE}"
 jinja2 ${INSERT_SQL} \
-   -D source=${SOURCE_TABLE//:/.} \
+   -D source_query=${SOURCE_QUERY//:/.} \
    -D vessel_info=${VESSEL_INFO//:/.} \
    -D named_anchorages=${NAMED_ANCHORAGES//:/.} \
    -D start_date=${START_DATE} \
    -D end_date=${END_DATE} \
-   | bq query --max_rows=0 --allow_large_results --replace --destination_table ${DEST_TABLE}
+   | bq query --max_rows=0 --allow_large_results --append_table --destination_table ${DEST_TABLE}
```

Comment:
@andres-arana why `append-table`? this will not make this bash idempotent. If you run it twice you will end up having duplicated entries?

Reply:
We are already deleting the records for the current date ranges beforehand in this same script. I don't know why we were using replace here.